{
  Throwable error=null;
  try {
    doSomething();
  }
 catch (  AssertionError e) {
    throw e;
  }
catch (  Throwable e) {
    error=e;
  }
  if (error != null) {
  }
}
